webui/src/Components/form.css

```
@@ -0,0 +1,248 @@
+		&.col-form-label-sm {
+			padding-top: calc(0.25rem + 1px);
+			padding-bottom: calc(0.25rem + 1px);
+			font-size: 0.875rem;
+		}
+	}
+
+	.feature-icons {
+		display: inline-flex;
+		align-items: center;
+
+		background-color: var(--color-border);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)